PLANNING.md

```diff
@@ -0,0 +1,70 @@
+# Mobile Team Releases Planning
+
+This document describes how the mobile team is planning each one of the releases.
+
+**What’s a release?**
+
+The release is every version of the app that’s sent to the store as a production release. The release is always a major/minor update, like from (1.0.0 to 1.1.0 or to 2.0.0). Patch releases (from 1.0.0 to 1.0.1) won’t follow this steps and are considered hotfixes releases. Our versioning is following the [Semantic Versioning 2.0.0](https://semver.org) guide.
+
+
+**What are the important days of a release?**
+
+The first release candidate (TestFlight and Beta) needs be done by 27th of each month. The release will always happen on day 5th of each month, unless there’s some critical crash/bug happening.
+
+
+**What happens if something could not be done in time for the release candidate?**
+
+In general, if it’s a new feature, it’ll be postpone for the next release only.
+
+
+**When do we plan the release features/improvements/bugs?**
+
+Every month can be a different day between 27th and 5th to plan the next release. The leader of the team will schedule the team and the all team will be able to participate of the planning. At this moment, most of the issues will be assigned to each member of the team.
```

Comment:
I think that it should be 'all the team' instead of 'the all team' and 'participate in' instead of 'participate of' in the second line.

Reply:
👍